docs/cody/capabilities/supported-models.mdx

```diff
@@ -0,0 +1,34 @@
+# Supported Large Language Models
+
+## Chat
+
+Cody supports a variety of cutting edge large language models for use in Chat, allowing you to select the best model for your use case.  Free users are defaulted to Claude 2.0 model from Anthropic, while Pro users have access to select any supported model.
+
+| **Provider** | **Model** | **Free** | **Pro** | **Enterprise** |
+|:--|:--|:--|:--|:--|
+| OpenAI | [gpt-3.5 turbo](https://platform.openai.com/docs/models/gpt-3-5-turbo) | - | ✅ | ✅ |
+| OpenAI | [gpt-4 turbo](https://platform.openai.com/docs/models/gpt-4-and-gpt-4-turbo#:~:text=TRAINING%20DATA-,gpt%2D4%2D0125%2Dpreview,-New%20GPT%2D4) | - | ✅ | ✅ |
+| Anthropic| [claude Instant](https://docs.anthropic.com/claude/docs/models-overview#model-comparison) | - | ✅ | ✅ |
+| Anthropic| [claude-2.0](https://docs.anthropic.com/claude/docs/models-overview#model-comparison) | ✅ | ✅ | ✅ |
+| Anthropic| [claude-2.1](https://docs.anthropic.com/claude/docs/models-overview#model-comparison) | - | ✅ | ✅ |
+| Anthropic| [claude-3 Haiku](https://docs.anthropic.com/claude/docs/models-overview#model-comparison) | - | ✅ | *coming soon* |
```

Comment:
We are changing default chat for both to be [sonnet](https://github.com/sourcegraph/cody/pull/3575) in the latest pr that you co-ordinated with philipp. 

<img width="560" alt="image" src="https://github.com/sourcegraph/docs/assets/11155207/6b72c8a4-1021-4755-a302-08cad1fc57cf">